qa-covid19.planx-pla.net/manifest.json

```
@@ -47,6 +47,7 @@
     "tier_access_level": "regular",
     "tier_access_limit": "2",
     "useryaml_s3path": "s3://cdis-gen3-users/qa-covid19/user.yaml",
+    "covid19_etl_bucket": "s3://static.planx-pla.net",
```

```suggestion
    "covid19_data_bucket": "s3://static.planx-pla.net",
```
suggesting this small change since the bucket can be used for more than just the ETLs